### PR TITLE
Add Universal Sentence Encoder QA baseline

### PR DESCRIPTION
**Pull request by matthen**
this is the dual encoder model from https://tfhub.dev/google/universal-sentence-encoder-multilingual-qa/1

### CODE REVIEW

File: baselines/run_baselines.sh [hunk 1]
```diff
@@ -11,13 +11,12 @@ OUTPUT_FILE=baselines/results.csv
 echo "method, train, test, train_size, test_size, recall_k, accuracy" > ${OUTPUT_FILE}
 
 for DATASET in reddit os amazon; do
-  for METHOD in TF_IDF BM25 USE_SIM USE_MAP USE_LARGE_SIM
-   USE_LARGE_MAP ELMO_SIM ELMO_MAP BERT_SMALL_SIM BERT_SMALL_MAP BERT_LARGE_SIM BERT_LARGE_MAP; do
+  for METHOD in USE_QA; do #TF_IDF BM25 USE_SIM USE_MAP USE_LARGE_SIM USE_LARGE_MAP ELMO_SIM ELMO_MAP BERT_SMALL_SIM BERT_SMALL_MAP BERT_LARGE_SIM BERT_LARGE_MAP USE_QA; do
```
Comment (pawel-polyai) [inline baselines/run_baselines.sh]: is this commenting out deliberate?

File: requirements.txt [hunk 2]
```diff
@@ -74,7 +74,8 @@ tensorflow==1.12.0
 tensorflow-estimator==1.10.12
 tensorflow-hub==0.3.0
 termcolor==1.1.0
+tf-sentencepiece==0.1.82
 tqdm==4.31.1
 typing==3.6.6
-urllib3==1.24.2
+urllib3==1.24.1
```
Comment (pawel-polyai) [inline requirements.txt]: why do we downgrade this?

Comment (matthen) [reply requirements.txt]: aha that was a mismatch with my machine, reverted

File: baselines/run_baselines.sh (hunk 1 above)
Comment (matthen) [reply baselines/run_baselines.sh]: nope! reverted!

**Pull request merged**